openedx/core/djangoapps/course_groups/views.py

```diff
@@ -90,7 +90,8 @@ def _get_cohort_representation(cohort, course):
     return {
         'name': cohort.name,
         'id': cohort.id,
-        'user_count': cohort.users.count(),
+        'user_count': cohort.users.filter(courseenrollment__course_id=course.location.course_key,
```

Comment:
@cahrens I added to an existing test.  Would you like me to move it to its own test?  This test fails on Master.